src/buttons/radio.ts

```diff
@@ -117,11 +132,17 @@ export class NgbRadio implements OnDestroy {
 
   get checked() { return this._checked; }
 
-  get disabled() { return this._disabled; }
+  get disabled() {
+    let result = (this._group && this._group.disabled) || this._disabled;
+    if (this._label) {
+      this._label.disabled = result;
```

Comment:
I'm not super-fan of having side-effects in a getter (in other words - we shouldn't be doing this). What is the motivation / problem that we are trying to solve here?